core/Makefile

```diff
@@ -503,6 +503,9 @@ upload_prodtest: ## upload prodtest using trezorctl
 coverage:  ## generate coverage report
 	./tools/coverage-report
 
+coverage_hits:  ## generate annotated source with per-line hits
+	./tools/coverage-annotate.py $(shell find . -name '.coverage*.json*') > htmlcov/hits.md
```

Comment:
I'd add this under the `coverage` target

Reply:
Fixed in https://github.com/trezor/trezor-firmware/pull/4962/commits/cceb66f45f93ed2ddb2548ced79a6b8b5c5704f0

Reply:
Oops, thanks!
Should be fixed in https://github.com/trezor/trezor-firmware/pull/4962/commits/f1fc7a02c0f3570c7182f2dc8e15adbd1a5cecbf.